python/pycylon/data/table.pyx

```diff
@@ -386,3 +386,14 @@ cdef class Table:
         py_arrow_table = pyarrow_wrap_table(table)
         return py_arrow_table.schema
 
+    def __getitem__(self, key):
+        table = to_pyarrow_table(self.id.encode())
+        py_arrow_table = pyarrow_wrap_table(table)
+        ctx = CylonContext(config=None)
```

Comment:
shouldn't we take ctx from the table itself? this reinitialization would be problematic, isn't it?

Reply:
at the moment we don't support the wrapping and unwrapping C++ vs Python objects. We need to get that to do it. 
I am still exploring how to do that.

Reply:
There is an easy fix in Python class to get the context when it is being built. That is the straightforward way to do it.

Reply:
I checked this. Without Python CPP Object Wrapping Unwrapping, this won't be feasible at the moment.

Reply:
Added an issue. 
https://github.com/cylondata/cylon/issues/172
Need to work on this separately for a major refactor.

Reply:
@supunkamburugamuve @nirandaperera for this PR, the context creation issue was fixed.